### PR TITLE
add cron dataload e2e test

### PR DESCRIPTION
**Pull request by wangshli**
### Ⅰ. Describe what this PR does
Add e2e test to verify cron dataload. 

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
test/prow/testcases/common/cron_dataload.py

### Ⅳ. Describe how to verify it
Run e2e test. 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: test/prow/testcases/common/cron_dataload.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+    try:
+        flow.run()
+    except Exception as e:
+        print(e)
+        exit(1)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (cheyang) [inline test/prow/testcases/common/cron_dataload.py:177]: Please add return of line.

File: test/prow/testcases/common/cron_dataload.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    )
+
+    flow.append_step(
+        StatusCheckStep(
+            step_name="check if the whole dataset is warmed up",
+            forth_fn=funcs.check_dataset_cached_percentage_fn(name, namespace)
+        )
+    )
+
+    flow.append_step(
```
Comment (cheyang) [inline test/prow/testcases/common/cron_dataload.py:154]: Please check the generated cronjob. Thanks.

Comment (wangshli) [reply test/prow/testcases/common/cron_dataload.py:154]: ok.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**